Makefile

```diff
@@ -40,11 +40,17 @@ test-e2e: GO_TEST_PACKAGES :=./test/e2e/...
 test-e2e: GO_TEST_FLAGS += -v
 test-e2e: test-unit
 
-update-codegen-crds:
-	go run ./vendor/github.com/openshift/library-go/cmd/crd-schema-gen/main.go --domain openshift.io --apis-dir vendor/github.com/openshift/api
+CRD_GOPATH :=$(shell mktemp -d)
```

Comment:
this will create a tmpdir on every invocation of the makefile even if nor related to crd, make it target specific